ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/WriteServiceWithDeduplicationSupportSpec.scala

```diff
@@ -81,7 +84,7 @@ class WriteServiceWithDeduplicationSupportSpec
         eqTo(Configuration.reasonableMaxDeduplicationTime),
         eqTo(LedgerTimeModel.reasonableDefault),
         eqTo(ApplicationId(submitterInfo.applicationId)),
-        eqTo(submitterInfo.actAs.toSet),
+        any[Set[Ref.Party]],
```

Comment:
Why not specify the exact value?

Reply:
I'll be honest: because I hate mockito.
I have 2 issues that have been plaguing me:

1. If somehow the arguments differ, we'll get the fabulous NPE and I always lose time till I figure it out
2. Due to mockito's 2 step design (first step, `when` - `then`, which sets the response....second step the verify which validate the call) we basically run the assertions twice, and it would always fail not at the verify but beforehand.  Therefore, if we expect a mock call I would expect a `verify` to be done, with the exact value. But when mocking to specify the return, I think it makes more sense to use exact value only if somehow we have multiple mocked responses which differ based on the values themselves.

Reply:
And the other problem with using the exact value here, is that it couples the tests.
Why should this test fail if we got different parties for readers? This unit test should not verify that as well.

Reply:
I'm convinced. Ship it.